documentation/modules/rn-2.6.adoc

```diff
@@ -2,6 +2,287 @@
+
+
+.Data loss is no longer possible when migrating VMware VMs with snapshots
+
+In the previous versions of {product-short}, when migrating a VMware VM that had a snapshot, the VM that was created in {ocp-name} Virtualization contained the data in the snapshot, but not in the running VM that was migrated. This has been resolved in {product-short} 2.6.0.  link:https://issues.redhat.com/browse/MTV-447[(MTV-447)]
+
+
+.Adding an OVA provider no longer displays the `ConnectionTestFailed` error message
+
+In the previous versions of {product-short}, when adding an OVA provider, `The provider is not ready - ConnectionTestFailed` error message appeared, even though the provider was eventually created. The message disappeared after a few seconds. This has been resolved in {product-short} 2.6.0. link:https://issues.redhat.com/browse/MTV-671[(MTV-671)]
+
+
+.Canceling and deleting a failed migration plan does not clean up the `populate` pods and PVC
+
+In earlier releases of {project-short}, when you canceled and deleted a failed migration plan, and after creating a PVC and spawning the `populate` pods, the `populate` pods and PVC were not deleted. You had to delete the pods and PVC manually. This issue has been resolved in {project-short} 2.6.0. link:https://issues.redhat.com/browse/MTV-678[(MTV-678)]
+
+
+.{ocp} to {ocp} migrations require the cluster version to be 4.13 or later
+
+In earlier releases of {project-short}, when migrating from {ocp} to {ocp}, the version of the source provider cluster had to be {ocp} version 4.13 or later. This issue has been resolved in {project-short} 2.6.0. link:https://issues.redhat.com/browse/MTV-809[(MTV-809)]
+
+
+.Multiple storage domains are always mapped to a single storage class
+
+In earlier releases of {project-short}, multiple disks from different storage domains were always mapped to single storage class. This issue has been resolved in {project-short} 2.6.0. link:https://issues.redhat.com/browse/MTV-1008[(MTV-1008)]
+
+
+.Firmware detection by virt-v2v
+
+In earlier releases of {project-short}, a `virt-v2v` that converted a guest also produced an OFV with configuration based on the virtual disk, such as driver or firmware. The `forklift-controller` processed the produced OFV's status of the conversion pod. This issue has been resolved in {project-short} 2.6.0. link:https://issues.redhat.com/browse/MTV-759[(MTV-759)]
+
+
+.Creating a host secret requires validation of the secret before creation of the host
+
+In earlier releases of {project-short}, when creating a host secret, you had to validate the secret before creating the host. This issue has been resolved in {project-short} 2.6.0. link:https://issues.redhat.com/browse/MTV-868[(MTV-868)]
+
+
+.When adding OVA provider a `ConnectionTestFailed` message appears
```

Comment:
it already appears in the resolved issues section (.Adding an OVA provider no longer displays the `ConnectionTestFailed` error message), this item can be dropped

Reply:
Deleted earlier entry, polished your text and changed